.orchestra/config/components/toolchain/lib/pkgconfig.lib.yml

```diff
@@ -0,0 +1,40 @@
+#@yaml/text-templated-strings
+
+#@ load("/lib/fn_args.lib.yml", "mandatory")
+#@ load("/lib/create_component.lib.yml", "single_build_component")
+
+#@yaml/text-templated-strings
+---
+#@ def pkgconfig_args(
+#@        triple=mandatory,
+#@ ):
+configure: |
+  mkdir -p "$BUILD_DIR"
+  # Nothing else to do
+install: |
+  mkdir -p "${TMP_ROOT}${ORCHESTRA_ROOT}/(@= triple @)/bin"
+  cat >> "${TMP_ROOT}${ORCHESTRA_ROOT}/(@= triple @)/bin/(@= triple @)-pkg-config" << EOF
+  #!/bin/bash
+  # Modified from a public-domain file taken from
+  # http://www.mega-nerd.com/erikd/Blog/CodeHacking/MinGWCross/pkg-config.html
+
+  # When using the mingw32msvc cross compiler tools, the native Linux
+  # pkg-config executable works fine as long as the default PKG_CONFIG_LIBDIR
+  # is overridden.
+  DIR="\$( cd "\$( dirname "\${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+  export PKG_CONFIG_LIBDIR="\$DIR/../lib/pkgconfig"
+
+  export PKG_CONFIG_PATH=\${PKG_CONFIG_LIBDIR}
+
+  # Now just execute pkg-config with the given command line args.
+  exec pkg-config \$@
+  EOF
```

Comment:
This script sounds like a black-box we don't what is doing, and it has some bugs (e.g., `$@` is not quoted).

I think the following is enough:
```bash
#!/bin/bash
DIR="\$( cd "\$( dirname "\${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
export PKG_CONFIG_PATH="\$DIR/../lib/pkgconfig"
exec pkg-config "\$@"
```